docs/for-devs/get-started/lab-10.md

```diff
@@ -1,16 +1,16 @@
 ---
 slug: lab-10
-title: BYO manifest to deploy a workload with Argo CD
-sidebar_label: Using Argo CD
+title: BYO manifests to create a workload
+sidebar_label: BYO manifests
 ---
 
 :::info
-Prerequisite: For this lab, Argo CD needs to be activated.
+Argo CD needs to be activated for this lab.
 :::
 
-Deploying your applications by doing `kubectl apply -f` is not ideal. You as a developer would like to automatically deploy and update your application after a new build. Otomi integrated Argo CD to provide an out-of-the-box gitops solution.
+Deploying your applications by doing `kubectl apply -f` is not ideal. You as a developer would like to automatically deploy and update your application after a new build. Otomi integrated Argo CD to provide an out-of-the-box GitOps solution.
```

Comment:
Perhaps it is better to say "usually developers need", instead of "you as developer would like"